<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverWindow</class>
 <widget class="QMainWindow" name="DriverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>司机端</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow {
    background-color: #f5f7fa;
    border: none;
   }
   
   QGroupBox {
    background-color: white;
    border-radius: 10px;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
    border: 1px solid #e8e8e8;
   }
   
   QPushButton {
    border-radius: 8px;
    padding: 10px;
    font-weight: bold;
    border: none;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
   }
   
   QPushButton:hover {
    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.15);
   }
   
   QPushButton:pressed {
    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
   }
   
   QProgressBar {
    border-radius: 6px;
    background-color: #f0f0f0;
    height: 12px;
   }
   
   QProgressBar::chunk {
    background-color: #52c41a;
    border-radius: 6px;
   }
   
   QLabel {
    color: #333333;
   }
   
   .titleLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
   }
   
   .valueLabel {
    font-size: 14px;
    color: #666666;
   }
   
   .statusOnline {
    color: #52c41a;
    font-weight: bold;
   }
   
   .statusOffline {
    color: #ff4d4f;
    font-weight: bold;
   }
   
   .statusBusy {
    color: #fa8c16;
    font-weight: bold;
   }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <property name="styleSheet">
       <string>QWidget {
        background-color: white;
        border-radius: 10px;
        box-shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
        padding: 15px;
       }</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="driverInfoLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>司机：李四 | 湘A12345 | 大众朗逸</string>
         </property>
         <property name="styleSheet">
          <string>color: #52c41a;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="onlineBtn">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: #52c41a; color: white;</string>
         </property>
         <property name="text">
          <string>上线</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>当前状态：离线</string>
         </property>
         <property name="styleSheet">
          <string>color: #ff4d4f;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="orderInfoGroup">
           <property name="title">
            <string>订单信息</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="orderIdLabel">
              <property name="text">
               <string>订单编号：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="passengerInfoLabel">
              <property name="text">
               <string>乘客：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="startLocationLabel">
              <property name="text">
               <string>起点：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="endLocationLabel">
              <property name="text">
               <string>终点：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="estimatedDistanceLabel">
              <property name="text">
               <string>预估距离：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="estimatedTimeLabel">
              <property name="text">
               <string>预估时长：</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="acceptOrderBtn">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string>background-color: #52c41a; color: white;</string>
                </property>
                <property name="text">
                 <string>接单</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rejectOrderBtn">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string>background-color: #ff4d4f; color: white;</string>
                </property>
                <property name="text">
                 <string>拒单</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="tripControlGroup">
           <property name="title">
            <string>行程控制</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="tripStatusLabel">
              <property name="text">
               <string>行程状态：未开始</string>
              </property>
              <property name="styleSheet">
               <string>color: #fa8c16; font-weight: bold;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="startTripBtn">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string>background-color: #00b2ff; color: white;</string>
                </property>
                <property name="text">
                 <string>开始行程</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="endTripBtn">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string>background-color: #fa8c16; color: white;</string>
                </property>
                <property name="text">
                 <string>结束行程</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="tripProgressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="drivenDistanceLabel">
              <property name="text">
               <string>已行驶里程：0.0 km</string>
              </property>
              <property name="styleSheet">
               <string>color: #666666;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <property name="styleSheet">
         <string>QWidget {
          background-color: white;
          border-radius: 10px;
          box-shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
        }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="navigationGroup">
           <property name="title">
            <string>导航地图</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QGraphicsView" name="mapView">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>600</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
